### ISSUE TITLE
FormDataSet.AsMultipart() first empty line problem

### ISSUE THREAD

**Issue by tsu1980** [AngleSharp/AngleSharp#118]
I uncountered problem that POST login form failed. server is IIS/7.5.

When i remove first empty line in http body(following code), problem disappeared. maybe this empty line unnecessary and bad effect for IIS?

``` cs
        public Stream AsMultipart(Encoding encoding = null)
        {
            encoding = encoding ?? TextEncoding.Utf8;
            var ms = new MemoryStream();
            CheckBoundaries(encoding);
            ReplaceCharset(encoding);
            var tw = new StreamWriter(ms, encoding);
            //tw.WriteLine();    // bad for IIS?
```

AngleSharp v0.8.8
use default HttpRequester

**Comment by FlorianRappl** [comment]
Hm that's weird because this method is somehow tested - and most tests are being performed against a site hosted on Azure, which presumably uses IIS (not sure of the version though, but that shouldn't really matter).

Unfortunately I can't find any reference of the initial empty line. The `AsUrlEncoded` enctype does not use it, but `AsPlaintext` does. All tests are also passing without the initial empty line. I therefore assume that the empty line is required (it is the header / body separator), but is added automatically by `HttpWebRequest`. 

The problem may therefore be related to redundant empty lines, but I'll continue investigate.

On which platform did you use AngleSharp [.NET4, SL, WP8.1, ...]?

**Comment by tsu1980** [comment]
Thank you for reply.

I use on .NET 4.5.1.

Yeah the problem is redundant empty lines, i think so too.

I compared two requests(AngleSharp, Chrome) in binary level. then i had reached this redundant empty lines.

I am not familiar with the standard specifications, so I don't understand which one is correct. Anyway, i will use source build version for a while :)

**Comment by FlorianRappl** [comment]
Will be fixed in the last release prior to v0.9. I'm quite sure that the redundant line is not specified and that I mixed up something.
